5.3_task.py

```diff
@@ -0,0 +1,12 @@
+import os
+os.chdir("./directory/for/file")
+with open ('text.txt', 'r') as file:
+    result = file.read()
+all_freq = {}
+for i in result:
+    if i in all_freq:
+        all_freq[i] += 1
+    else:
+        all_freq[i] = 1
```

Comment:
what is i?

also with this approach, you will calculate all numbers and symbols. But I want only letters